prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1392,7 +1392,12 @@ static inline int rxr_ep_send_queued_pkts(struct rxr_ep *ep,
 			dlist_remove(&pkt_entry->entry);
 			continue;
 		}
-		ret = rxr_pkt_entry_send(ep, pkt_entry, pkt_entry->addr);
+		if (pkt_entry->send->iov_count > 0)
```

Comment:
Can we move the refactor to the first patch?

Reply:
Will do